src/main/webapp/app/overview/course-faq/course-faq.component.ts

```diff
@@ -26,11 +28,13 @@ import { ArtemisMarkdownModule } from 'app/shared/markdown.module';
     standalone: true,
     imports: [ArtemisSharedComponentModule, ArtemisSharedModule, CourseFaqAccordionComponent, CustomExerciseCategoryBadgeComponent, SearchFilterComponent, ArtemisMarkdownModule],
 })
-export class CourseFaqComponent implements OnInit, OnDestroy {
+export class CourseFaqComponent implements OnInit, OnDestroy, AfterViewInit {
+    @ViewChildren('faqElement') faqElements!: QueryList<ElementRef>;
```

Comment:
We should ideally use [viewChildren](https://angular.dev/api/core/viewChildren?tab=usage-notes) here to comply with the client coding guidelines